modules/yaml-config/main.tf

```diff
@@ -83,18 +92,26 @@ data "template_file" "remote_config" {
   vars     = var.parameters
 }
 
+module "deep_merge" {
+  source = "../deepmerge"
+  maps   = concat([{}], local.remote_map_configs_decoded, local.local_map_configs_decoded)
+  #  maps           = concat(local.remote_map_configs, local.local_map_configs)
```

Comment:
Why comment this out instead of removing ?

Reply:
still testing things lol